doc/src/site/sphinx/deployment/backends.rst

```diff
@@ -144,11 +117,11 @@ environment at the moment. We recommend this approach, as it is easier to deploy
 and it is also more suitable for production environments. When the H2O cluster is started on YARN, it is started
 as a map reduce job, and it always uses the flatfile approach for nodes to cloud up.
 
-Get extended H2O driver, for example, for cdh 5.8:
+Get H2O driver, for example, for cdh 5.8:
 
 .. code:: bash
 
-    H2O_EXTENDED_JAR=$(./bin/get-extended-h2o.sh cdh5.8)
+    H2O_DRIVER_JAR=$(./bin/get-h2o-driver.sh cdh5.8 | tail -n 1 | awk 'NF>1{print $NF})
```

Comment:
Is this part necessary? `| tail -n 1 | awk 'NF>1{print $NF})` I would prefer fixing the `./bin/get-h2o-driver.sh` and keep the user-facing command simple, WDYT?

Reply:
Np, will do.